app.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
```

Comment:
There's no need to have app.use(express.json()) at this stage - please remove it.